src/resolve/resolved-dns-cache.c

```diff
@@ -1024,6 +1043,15 @@ int dns_cache_lookup(
                         goto miss;
                 }
 
+                /* Skip the next part if ttl is expired in first attempt*/
+                if( !use_stale_data && j->ttl_expiry < current){
```

Comment:
coding style, space *before* the "(", not after.

and space between ) and {

Reply:
Fixed